<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientFormDialog</class>
 <widget class="QDialog" name="PatientFormDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PatientFormDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>241</width>
       <height>261</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <widget class="LineEdit" name="addressEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>50</number>
      </property>
     </widget>
     <widget class="ComboBox" name="sexCombo">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>190</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maxVisibleItems">
       <number>2</number>
      </property>
      <item>
       <property name="text">
        <string>Male</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Female</string>
       </property>
      </item>
     </widget>
     <widget class="DateEdit" name="birthEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>160</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="maximumDateTime">
       <datetime>
        <hour>9</hour>
        <minute>59</minute>
        <second>59</second>
        <year>9999</year>
        <month>12</month>
        <day>25</day>
       </datetime>
      </property>
      <property name="minimumDate">
       <date>
        <year>1900</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>1900</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
     <widget class="QLabel" name="addressLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Address:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Sex:</string>
      </property>
     </widget>
     <widget class="NameEdit" name="lNameEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maxLength">
       <number>50</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Phone:</string>
      </property>
     </widget>
     <widget class="LineEdit" name="phoneEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>39</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>First name:</string>
      </property>
     </widget>
     <widget class="NameEdit" name="fNameEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maxLength">
       <number>50</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Last name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Birth:</string>
      </property>
     </widget>
     <widget class="ColorPicker" name="colorPicker">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>220</y>
        <width>31</width>
        <height>32</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Избор на цвят</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Color code:</string>
      </property>
     </widget>
     <widget class="NameEdit" name="idLineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maxLength">
       <number>50</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifier:</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="errorLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">color : red;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>View/uiComponents/LineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>DateEdit</class>
   <extends>QDateEdit</extends>
   <header>View/uiComponents/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>NameEdit</class>
   <extends>QLineEdit</extends>
   <header>View/uiComponents/NameEdit.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>View/uiComponents/ComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ColorPicker</class>
   <extends>QPushButton</extends>
   <header>View/uiComponents/ColorPicker.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>idLineEdit</tabstop>
  <tabstop>fNameEdit</tabstop>
  <tabstop>lNameEdit</tabstop>
  <tabstop>phoneEdit</tabstop>
  <tabstop>addressEdit</tabstop>
  <tabstop>birthEdit</tabstop>
  <tabstop>sexCombo</tabstop>
  <tabstop>colorPicker</tabstop>
  <tabstop>okButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
